linuxbrew-build.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+ABS_PATH_LIMIT=85
+export HOMEBREW_NO_AUTO_UPDATE=1
+
+[[ -x ./bin/brew ]] || (echo "This script should be run inside Linuxbrew directory."; exit 1)
+
+cd $(realpath .)
```

Comment:
This needs quotes around the `cd` argument:
```suggestion
cd "$(realpath .)"
```
Test:
```
$ mkdir a\ b
$ cd $(echo "a b")
bash: cd: too many arguments
```